oneanddone/tasks/templates/tasks/activity_listing.html

```diff
@@ -24,6 +24,15 @@
           <div>{{ _("Team: {team}")|fe(team=attempt.task.team.name) }}</div>
           <div>{{ _('Date {state}: {modified}')|fe(state=attempt.get_state_display(),
             modified=attempt.modified.strftime('%Y-%m-%d: %I:%M:%S %p').lower()) }}</div>
+          {% if attempt.has_feedback %}
```

Comment:
I believe that the code you wrote above for `attempt.time_spent_in_minutes` will return `None` if `attempt.has_feedback` is `False`, so I think you could just use `{% if attempt.time_spent_in_minutes %}` and not even bother checking `if attempt.has_feedback`.